app/editor/src/features/reports/av-overview/AVOverview.tsx

```diff
@@ -76,7 +76,9 @@ export const AVOverview: React.FC = () => {
       <FormikForm
         initialValues={instance}
         onSubmit={(values, { setSubmitting }) => {
-          handleSubmit(values);
+          handleSubmit(values)
+            .catch((err) => toast.warning(() => 'Failed to save.'))
```

Comment:
1) Why a warning when it's an error?
2) Why doesn't the default toast work when submitting?  It should already be toasting the error message?